hooks/useCohort/useCohort.tsx

```diff
@@ -0,0 +1,58 @@
+import { createContext, useState, useEffect, useMemo, useContext } from "react";
+import { ApiServiceBuilder } from "@/helpers/api";
+import { HTTP_METHOD } from "@/types/api";
+import {
+  ICohort,
+  CohortProviderProps,
+  DEFAULT_COHORT_YEAR,
+} from "@/types/useCohort";
+
+const CohortContext = createContext<ICohort>({
+  currentCohortYear: DEFAULT_COHORT_YEAR,
```

Comment:
default should be null because what if the year formats are changed by admins to '22' instead of '2022', then the default would be wrong